### PR TITLE
8.0 imp mass mailing distribution list jne

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
Use the mail compose message to compute the distribution list with the `dl_computed`

### CODE REVIEW

File: distribution_list/distribution_list.py [hunk 1]
```diff
@@ -414,6 +414,9 @@ def _get_record(self, record_or_list):
         self.pool.get('res.company')._company_default_get(
             cr, uid, 'distribution.list.line', context=c),
         'domain': "[]",
+        'src_model_id': lambda self, cr, uid, c:
+            self.pool.get('ir.model').search(
+            cr, uid, [('model', '=', 'res.partner')], context=c)[0],
```
Comment (Cedric-Pigeon) [inline distribution_list/distribution_list.py]: Better use <code>limit=1</code> if you only take the first.

File: mass_mailing_distribution_list/tests/test_mail_compose_message.py [hunk 2]
```diff
@@ -37,8 +39,7 @@ def setUp(self):
         self.mass_mailing_obj = self.registry['mail.mass_mailing']
         self.mail_compose_message_obj = self.registry['mail.compose.message']
         self.dst_model_id = self.registry('ir.model').search(
-            self.cr, self.uid, [('model', '=',
-                                 'mail.mass_mailing.contact')])[0]
+            self.cr, self.uid, [('model', '=', SRC_MODEL)])[0]
```
Comment (Cedric-Pigeon) [inline mass_mailing_distribution_list/tests/test_mail_compose_message.py]: Same remark, use <code>limit=1</code>

**Comment by Olivier-LAURENT** [comment]
@Cedric-Pigeon `limit=1` done !

**Comment by Cedric-Pigeon** [comment]
:+1:

**Comment by adrienpeiffer** [comment]
:+1: (Code review)

**Pull request merged**